gandy/tests/test_models/test_gans.py

```diff
@@ -0,0 +1,200 @@
+"""Testing functions for UncertaintyModel gan class."""
+
+# import numpy as np
+import unittest
+import unittest.mock as mock
+
+# import deepchem
+
+import gandy.models.gans as gans
+import gandy.models.models
+
+
+class TestGAN(unittest.TestCase):
+    """Test GAN class."""
+
+    def test_inheritence():
+        """Ensure the subclass class inherits from parent class."""
+        assert issubclass(gans.gan, gandy.models.models.UncertaintyModel)
```

Comment:
It will yell without self as method input. Also, `self.assertTrue(issubclass...)` will give better error message than raw assert statement